test/load-balancer-mocha.js

```diff
@@ -0,0 +1,90 @@
+// load-balancer-mocha.js
+
+var chai = require('chai');
+var chaiAsPromised = require('chai-as-promised');
+chai.should();
+chai.use(chaiAsPromised);
+
+var http = require('http');
+var _ = require('underscore');
+var os = require('os');
+var fs = require('fs');
+var path = require('path');
+var noflo = require('noflo');
+var test = require('./common-test');
+
+describe('load-balancer', function() {
+    var port = 1337;
+    it("should POST input to single target", function() {
+        return new Promise(function(done, fail){
```

Comment:
I'm only running it on my machine, so it hard to say how long it will take with different hardware. If we had a build machine things would be different.